test/input_json/FinancialModels/communitysolar/2022.08.08_develop_branch_PVWatts_Community_Solar_cmod_communitysolar_outputs.json

```diff
@@ -523,8 +523,8 @@
 	"cf_community_solar_recurring_generation" : [ 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 ],
     "community_solar_upfront_cost" : 70000,
     "community_solar_upfront_revenue" : 0,
-    "subscriber1_npv" : 7954940.5535759041,
-    "subscriber2_npv" : 3158303.7010947196,
+    "subscriber1_npv" : 923830.933,
```

Comment:
Please note that any dependent outputs in the json file will now not be valid if compared. See email sent 11/10/2022